### PR TITLE
Added general swizzle conversion param

### PR DESCRIPTION
**Pull request by slimbuck**
We would like to pack normal maps using GGGR which is more standard than the currently supported RRRG (using -separate_rg_to_color_alpha flag).

Added a general swizzle operator for maximum happiness.

### CODE REVIEW

File: basisu_tool.cpp [hunk 1]
```diff
@@ -85,6 +85,7 @@ static void print_usage()
 		" -no_alpha: Always output non-alpha basis files, even if one or more inputs has alpha\n"
 		" -force_alpha: Always output alpha basis files, even if no inputs has alpha\n"
 		" -separate_rg_to_color_alpha: Separate input R and G channels to RGB and A (for tangent space XY normal maps)\n"
+		" -swizzle rgba: Apply a general swizzle operation to the incoming colour channels\n"
```
Comment (willeastcott) [inline basisu_tool.cpp]: You probably ought to use American spelling here.

Comment (slimbuck) [reply basisu_tool.cpp]: Good point, done

**Comment by richgel999** [comment]
This looks great - I'll merge this after the UASTC update which is coming out in a few hours today.

**Comment by richgel999** [comment]
I'll manually merge this today.

**Comment by slimbuck** [comment]
Hi @richgel999,

Just wanted to follow up on this one.

Have you decided against merging this swizzle PR or it is still planned?

Thanks

**Comment by slimbuck** [comment]
closing this PR, will submit a new PR with clean history.

**Pull request closed without merging**